lib/DDG/Spice/GetEvents/Nearby.pm

```diff
@@ -1,15 +1,15 @@
-package DDG::Spice::GetEvents;
+package DDG::Spice::GetEvents::Nearby;
```

Comment:
@Uriger I'm wondering now if we should simply remove this IA altogether, and pass along the user's city to the other API endpoint when the query is "events near me", "local events", etc?